website/docs.md

```diff
@@ -746,7 +752,7 @@ fooLogger = logger.With("foo", "bar")
 fooLogger.Info("A log with attributes.")  // adds foo="bar"
 ```
 
-Note that you can also add normal print statements to your code. These prints
+**Note** that you can also add normal print statements to your code. These prints
```

Comment:
Can you do the same format for these, which should be:

**Note**: You can also...

Same below.